packages/backend/src/index.ts

```diff
@@ -124,6 +126,7 @@ async function main() {
   const service = createServiceBuilder(module)
     .loadConfig(config)
     .addRouter('', await healthcheck(healthcheckEnv))
+    .addRouter('', metricsHandler())
```

Comment:
I find this quite interesting! Pardon me if a silly question 😅 – how is `''` as a route different from

```suggestion
    .addRouter(metricsHandler())
```

?

Reply:
This is not an Express router, it's our builder class. https://github.com/backstage/backstage/blob/6d7126c2a1db0a50393c87a88b05f5dcbcc9212f/packages/backend-common/src/service/types.ts?plain=1#L24

Reply:
Both args are required, so the first one compiles and the second one doesn't 😅. Now why the single arg version doesn't exist is a different question :p

Reply:
Ah, thanks for that both of y'all! I thought it implemented the Express API